### PR TITLE
APP-837 / APP-838

### PR DESCRIPTION
**Pull request by robson-sanchez**
- APP-837 Remove Symphony dependencies from sdk
- APP-838 Remove Symphony dependencies from webcommons

### CODE REVIEW

File: integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/client/model/Envelope.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package org.symphonyoss.integration.provisioning.client.model;
+
+/**
+ * Wrap the data into an envelope.
```
Comment (ecarrenho-daitan) [inline integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/client/model/Envelope.java]: Add information here to explain that the Apps Repository API wraps all request and response content in a "data" object and that is the purpose of this wrapper class.

**Review by ecarrenho-daitan — COMMENTED** [review]
LGTM.

**Comment by ecarrenho-daitan** [comment]
@dnsymphony please review.

File: integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/client/AppRepositoryClient.java [hunk 2]
```diff
@@ -62,133 +48,91 @@
...
-  private IPersister authPersister;
+  private SymphonyHttpApiClient client;
 
-  @PostConstruct
-  public void init() {
-    SymphonyClientConfig config = new SymphonyClientConfig();
-    config.setKeymanagerUrl(properties.getKeyManagerUrl());
-    config.setSymphonyUrl(properties.getSymphonyUrl());
-    config.setLoginUrl(properties.getLoginUrl());
+  public List getAppsAvailable(String userId) throws AppRepositoryClientException {
```
Comment (dnsymphony) [inline integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/client/AppRepositoryClient.java]: Please add javadoc to all methods

**Review by dnsymphony — CHANGES_REQUESTED** [review]
Unit tests?

File: integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/client/model/Envelope.java (hunk 1 above)
Comment (robson-sanchez) [reply integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/client/model/Envelope.java]: FIXED

File: integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/client/AppRepositoryClient.java (hunk 2 above)
Comment (robson-sanchez) [reply integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/client/AppRepositoryClient.java]: FIXED

**Comment by robson-sanchez** [comment]
@dnsymphony I addressed the PR's comments and also added some unit tests to the module integration-provisioning.

The missing unit tests will be implemented in my next PR related to the APP-836

Please check again if everything is ok.

**Review by dnsymphony — APPROVED** [review]

**Comment by robson-sanchez** [comment]
I'm gonna open another PR directly on the symphonyoss

**Pull request closed without merging**